hw/chisel/src/test/scala/snax/streamer/StreamerTopGenerate.scala

```diff
@@ -0,0 +1,16 @@
+package snax.streamer
+
+import chisel3._
+import org.scalatest.flatspec.AnyFlatSpec
+import chiseltest._
+
+class StreamerTopGenerate
+extends AnyFlatSpec {
+
+  emitVerilog(
+    new StreamerTop(TestParameters.streamer),
+    Array("--target-dir", "generated/")
+  )    
+
+}
```

Comment:
I vote to put the parameter example for instancing gemm and simd back. We can write some comments in the code to say this example configuration is for which accelerator. Because I think 1) these example configurations are good for people to understand how to configure the streamer as the configuration format of the streamer is quite complicated. 2) making sure the streamer still works after any modification for these configurations is essential. 3) These configurations show the streamer's reusability, working for different accelerators.

Reply:
Alright! As discussed: we like to avoid having code that is not used anywhere. Therefore the example configuration is displayed in the added documentation. The tests for different configurations are run through the automatic generation of scala parameters using the .hjson configuration files! For now this is only ALU but when the other accelerators are added these will be tested as well!